scss/bitstyles/organisms/page-header/PageHeader.js

```diff
@@ -0,0 +1,156 @@
+import icons from '../../../../assets/images/icons.svg';
+import Button from '../../atoms/button/Button';
+
+const PageHeader = ({
+  topLeft,
+  topRight,
+  centerLeft,
+  centerRight,
+  bottomLeft,
+  bottomRight,
+}) => {
+  const pageHeader = document.createElement('header');
+  pageHeader.classList.add('o-page-header');
+  const content = document.createElement('div');
+  content.classList.add('a-content', 'o-page-header__content');
+
+  pageHeader.appendChild(content);
+
+  const contentPositions = [
+    { position: 'top-left', content: topLeft },
+    { position: 'top-right', content: topRight },
+    { position: 'center-left', content: centerLeft },
+    { position: 'center-right', content: centerRight },
+    { position: 'bottom-left', content: bottomLeft },
+    { position: 'bottom-right', content: bottomRight },
+  ];
+
+  contentPositions.forEach((item) => {
+    if (item) {
```

Comment:
Is there ever a case where there’s no item? (I think `forEach` wouldn’t iterate over e.g. en empty array). Maybe this could become the check for item.content, so we avoid empty divs?

```suggestion
    if (item.content) {
```